views/apartments/show.erb

```diff
@@ -1 +1,2 @@
 This is info about apartment #3.
+<h1> Apartment: <%= @apartment %></h1>
```

Comment:
how can we use attr_accessors we get from active record, to get different column values from our database?